terra-jupyter-aou/Dockerfile

```diff
@@ -264,8 +261,6 @@ RUN curl -L http://www.repeatmasker.org/RepeatMasker/RepeatMasker-4.1.4.tar.gz \
 RUN rm -f /usr/local/bin/workspace_cromwell.py
 # alias cromshell_beta to cromshell
 RUN echo "alias cromshell='cromshell-beta'" >> ~/.bash_aliases
-# Downgrade jupyter_contrib_nbextensions as the newer version breaks snippet_menu extension
-RUN pip install --force-reinstall -v "jupyter_contrib_nbextensions==0.5.1"
```

Comment:
downgrading this broke nbconvert

Reply:
You are saying that there is no need to downgrade anymore?

Reply:
as far as I know? It built fine, but I was going to maybe ask Yonghao about how to test that this extension is working when testing